shuttleofx_client/views.py

```diff
@@ -3,247 +3,281 @@
 import requests
 from functools import wraps
 from flask import (
-    request,
-    jsonify,
-    render_template,
-    abort,
-    Response,
-    redirect,
-    url_for,
-    session
+	request,
```

Comment:
We use an indentation of 4 spaces and no "tab", as defined in the python standard convention pep8.